sunpy/net/jsoc/jsoc.py

```diff
@@ -901,3 +901,71 @@ def _can_handle_query(cls, *query):
     @classmethod
     def _attrs_module(cls):
         return 'jsoc', 'sunpy.net.jsoc.attrs'
+
+    @classmethod
+    def register_values(cls):
+        # We always use the local file for now.
+        return cls.get_jsoc_values(load=True)
+
+    @staticmethod
+    def get_jsoc_values(load=False, save=False):
```

Comment:
should save and load be different methods here?

Reply:
Fixed.